app/views/spree/admin/bolt/_configuration.html.erb

```diff
@@ -0,0 +1,35 @@
+<% if @bolt_configuration.present? %>
+  <table class="index">
+    <thead>
+      <tr>
+        <th><%= SolidusBolt::BoltConfiguration.human_attribute_name(:merchant_public_id) %></th>
+        <th><%= SolidusBolt::BoltConfiguration.human_attribute_name(:environment_url) %></th>
+        <th><%= SolidusBolt::BoltConfiguration.human_attribute_name(:created_at) %></th>
+        <th><%= SolidusBolt::BoltConfiguration.human_attribute_name(:updated_at) %></th>
+        <th class="actions"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><%= @bolt_configuration.merchant_public_id %></td>
+        <td><%= @bolt_configuration.environment_url %></td>
+        <td>
+          <%= @bolt_configuration.created_at.to_s(:long) %>
+        </td>
+        <td>
+          <%= @bolt_configuration.updated_at.to_s(:long) %>
+        </td>
+        <td class="actions">
+          <% if can?(:edit, @bolt_configuration) %>
+            <%= link_to 'Edit', edit_admin_bolt_path(@bolt_configuration), class: 'btn btn-primary' %>
+          <% end %>
+          <% if can?(:destroy, @bolt_configuration) %>
+            <%= link_to 'Destroy', admin_bolt_path(@bolt_configuration), data: {:confirm => 'Are you sure?'}, method: :delete, class: 'btn btn-danger' %>
+          <% end %>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+<% else %>
+  <p> No Bolt Configuration found </p>
```

Comment:
since you are implementing locales I think it would be good to add them here too

Reply:
What should I add to the locale for this partial?

I didn't find anything that I can add to the locale for this partial.

Reply:
I thought maybe the names of the actions (`Edit`, `Destroy`) and the notice when there is no configuration found

Reply:
Done